scripts/Ballot/delegateVote.ts

```diff
@@ -0,0 +1,43 @@
+import { Contract, ethers } from "ethers";
+import "dotenv/config";
+import * as ballotJson from "../../artifacts/contracts/Ballot.sol/Ballot.json";
+import { Ballot } from "../../typechain";
+import { getWallet } from "../../lib/wallet";
+
+async function main() {
+  const wallet = getWallet();
+  console.log(`Using address ${wallet.address}`);
+  const provider = new ethers.providers.InfuraProvider(
+    "ropsten",
+    process.env.ROPSTEN_URL
+  );
```

Comment:
I created a helper function to replace this, also I please change your ROPSTEN_URL env name to INFURA_API_KEY as this is what it requires

Reply:
Its ok